python/semantic_kernel/connectors/ai/open_ai/open_ai_response.py

```diff
@@ -0,0 +1,210 @@
+from typing import AsyncGenerator, Dict, List, Optional, Union
```

Comment:
nit: missing header: # Copyright (c) Microsoft. All rights reserved.

Reply:
Could we automate adding this header in VSCode? Let me check how that's handled in BAP, we have it like that here.